kafka-streams-serdes/src/main/java/org/hypertrace/core/kafkastreams/framework/serdes/proto/ProtoDeserializer.java

```diff
@@ -0,0 +1,61 @@
+package org.hypertrace.core.kafkastreams.framework.serdes.proto;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import com.google.protobuf.Parser;
+import org.apache.kafka.common.serialization.Deserializer;
+
+/**
+ * Custom Proto Deserializer for Kafka.
+ *
+ * <p>This class provides a deserialization mechanism for Kafka messages using Protocol Buffers
+ * without schema validation. It extends the Kafka Deserializer interface and allows for direct
+ * deserialization of byte arrays into Proto message objects by utilizing the provided Parser for
+ * the specific Proto message type.
+ *
+ * <p>Motivation: In setups where both producers and consumers use the same Proto schemas, the need
+ * for schema validation becomes redundant. The built-in {@code kafkaProtoSerdes} from Confluent
```

Comment:
Producers and consumers may or may not be having the same proto. schema. But, one may be backward compatible with the other.
```suggestion
 * <p>Motivation: Since the proto. configurations are usually shared between the producer and the consumers, the field descriptors are well-known to both the parties. In cases when there are other mechanisms to validate proto. compatibilities, 
 * schema validation becomes redundant and this class can be used in such cases. The built-in {@code kafkaProtoSerdes} from Confluent
```